app/decorators/member_decorator.rb

```diff
@@ -0,0 +1,19 @@
+class MemberDecorator
+  def initialize(member)
+    @member = member
+  end
+
+  delefate :birthday, to: member
+
+  def formatted_birthday
+    if birthday.present?
+      birthday.strftime("%m/%d/%Y")
+    else
+      ""
+    end
```

Comment:
I like `birthday.present? : birthday.strftime("%m/%d/%Y") : ""` but I don't know how the team/rubocop feel about it.

Reply:
I am fine with that, not my usual style but nothing wrong with it and rubocop does not complain. anyone else have an opinion?